games_registry.py

```
@@ -0,0 +1,841 @@
+    previous = runtime.BASE_DIR
+    runtime._apply_game_root(work_dir)
+    try:
+        yield
+    finally:
+        runtime._apply_game_root(previous)
+
+
+def _doctor_layout_snapshot(game_root: str, has_tl: bool) -> tuple[str, str]:
+    try:
+        import gemini_translate_batch as batch_mod
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)